pkg/tcpip/iptables/targets.go

```diff
@@ -33,3 +33,11 @@ type UnconditionalDropTarget struct{}
 func (UnconditionalDropTarget) Action(packet buffer.VectorisedView) (Verdict, string) {
 	return Drop, ""
 }
+
+// PanicTarget just panics. It represents a target that should be unreachable.
+type PanicTarget struct{}
```

Comment:
Is this only used in tests? If so, can we say so.  It seems dangerous.

Reply:
It's not only used in tests. I've updated it to more closely follow what Linux does, which is print an error and drop the packet.